tests/cli/test_get_set.py

```diff
@@ -479,15 +479,17 @@ def test_set_delete_nm_passthrough(self):
           networkmanager:
             name: "myid with spaces"
             passthrough:
-              connection.permissions: ""
-              ipv4.dns-search: ""''')
+              connection:
+                permissions: ""
+              ipv4:
+                dns-search: ""''')
         ap_key = 'network.wifis.wlan0.access-points.SOME-SSID'
-        self._set([ap_key+'.networkmanager.passthrough.connection\\.permissions=null'])
+        self._set([ap_key+'.networkmanager.passthrough.connection.permissions=null'])
```

Comment:
question: Do we need to change the escaping in existing `netplan get` calls, out in the wild? That might have the potential to break the snapd integration used in Ubuntu Core.

Reply:
Yes, that (the escaping with `\`) will not work anymore if you are using get and set with settings inside the `passthrough` section. We probably shouldn't backport this change to LTS releases (like, never), even if we don't give any guarantees about stability for `passthrough`  block.